indigo_app/notifications.py

```diff
@@ -51,3 +68,37 @@ def post_save_task(sender, instance, **kwargs):
     if kwargs['created']:
         if isinstance(instance.action_object, Task):
             notifier.send_task_notifications(instance)
+
+
+@receiver(comment_was_posted, sender=Comment)
+def post_comment_save_notification(sender, **kwargs):
+    """
+    Send email when a user comments on a task
+    """
+    task_content_type = ContentType.objects.get_for_model(Task)
```

Comment:
For safety, please do an early check that the content type on the comment matches the Task's content type. Otherwise we're going to be confused in a few months when we add comments for things other than tasks :)